lib/json_rails_logger/json_formatter.rb

```diff
@@ -45,6 +49,24 @@ def process_message(raw_msg)
       { message: msg.strip }
     end
 
+    def format_message(msg)
+      new_msg = { rails: { environment: ::Rails.env } }
+
+      return msg.merge(new_msg) if msg.is_a?(Hash) &&
```

Comment:
The condition for the `if` would be better refactored into a method with a name that more clearly reveals the condition you are testing for